### PR TITLE
primo_sprinto into master

### PR DESCRIPTION
**Pull request by acancelliere**
Changes:
- Made less invasive the ringing on the incoming call during a GSM call
- kept last presence group selected
- Added a warning message during logout
- Added the ability to select the application to call numbers from the browser (not supported with Samsung browser)
- Fixed dark theme on presence
- Removed the word GROUPS and leaves only the name of the group in Presence
- Presence section added
- Added the ability to disconnect the app through a specific presence status
- Added the link in the Dashboard to the new Presence section

### CODE REVIEW

**Review by gsanchietti — COMMENTED** [review]
Thank you for pushing this.

The app is still on production, so it doesn't make sense a real review of the code right now.
From a formal point of view, the commit history is a little embarrassing, but I really appreciate the effort about an English recap inside the PR description.

If @SebastianMB-IT approves it, I'd go with a squash&merge to discard all useless commit messages.

**Comment by acancelliere** [comment]
Thanks for your feedback @gsanchietti, we will write better commit messages.
We wait for Sebastian's review and after that we will do a Squash&Merge.

**Review by 61f7b7437262085c97876892ddf9c0d8 — APPROVED** [review]

**Pull request merged**
